BuildAssets/test-linux-package-in-dists.sh

```diff
@@ -14,13 +14,18 @@ if [[ -z "$OCTOPUS_CLI_SERVER" || -z "$OCTOPUS_CLI_API_KEY" || -z "$OCTOPUS_SPAC
   exit 1
 fi
 
+export PKG_PATH_PREFIX="octopuscli"
+if [ ! -e "$PKG_PATH_PREFIX"*amd64.deb ]; then
```

Comment:
Add some validation to make this script easier to debug/diagnose - it took me a while to figure out how to configure things